src/lib/viewers/doc/_docBase.scss

```diff
@@ -36,9 +36,8 @@ $thumbnail-border-radius: 4px;
     }
 
     .bp-thumbnail-image {
-        border-radius: $thumbnail-border-radius;
+        object-fit: contain;
```

Comment:
What's the experience like in IE11? https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit#Browser_compatibility

Reply:
updating PR to use `background-image`